style.css

```diff
@@ -66,5 +66,17 @@ div.navbar-collapse > div.navbar-nav > a.nav-link:hover {
 .foot-link {
 	margin-right: 1.25rem;
 }
+#image{
+	height: 15rem;
+	width: auto;
+	margin-top: 80px;
+}
+
+#box{
+	height: 500px;
+	width: 500px;
+	border: 3px solid;
+
+}
 
 /* END: navbar/footer stuff */
```

Comment:
Respect this comment, shift your `#image {}` and `#box {}` to after the ` /* END: navbar/footer stuff */` comment for organisation

Reply:
-_-